packages/venia-concept/src/components/MyAccountMenuPage/constants.js

```diff
@@ -0,0 +1,8 @@
+import PropTypes from 'prop-types';
+
+export const USER_PROP_TYPES = {
+    email: PropTypes.string,
+    firstname: PropTypes.string,
+    lastname: PropTypes.string,
+    fullname: PropTypes.string
+};
```

Comment:
Make sure to delete this file and move the prop types into the component declaration, as we do in every other component.